package.json

```diff
@@ -15,7 +15,6 @@
     "modular": "ts-node packages/modular-scripts/src/cli.ts",
     "test": "yarn modular test --watchAll false --runInBand --forceExit",
     "build": "yarn workspace create-modular-react-app build && yarn workspace modular-scripts build && yarn workspace modular-views.macro build",
-    "release": "yarn build && changeset publish",
```

Comment:
Won't this allow people to publish in error without having built?

Reply:
I think only the publishing account we created has access to publish - we shouldn't be publishing locally anyway.